app/validate_txs.go

```diff
@@ -44,12 +45,20 @@ func FilterTxs(logger log.Logger, ctx sdk.Context, handler sdk.AnteHandler, txCo
 // function used to apply the ante handler.
 func filterStdTxs(logger log.Logger, dec sdk.TxDecoder, ctx sdk.Context, handler sdk.AnteHandler, txs [][]byte) ([][]byte, sdk.Context) {
 	n := 0
+	sdkTransactionsCount := 0
 	for _, tx := range txs {
 		sdkTx, err := dec(tx)
 		if err != nil {
 			logger.Error("decoding already checked transaction", "tx", tmbytes.HexBytes(coretypes.Tx(tx).Hash()), "error", err)
 			continue
 		}
+		msgTypes := msgTypes(sdkTx)
+		if sdkTransactionsCount+len(sdkTx.GetMsgs()) > v3consts.SdkMsgTransactionCap {
```

Comment:
this is making the assumption that we can't mix PFBs and sdk messages in the same transaction. I guess that's the case currently if I am not missing anything